sentryflow/config/default.yaml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Authors of SentryFlow
+
+filters:
+  server:
+    port: 8081
+
+  envoy:
+    uri: anuragrajawat/httpfilter:v0.1
```

Comment:
Do we need to change images from personal repo to, maybe 5gsec/httpfilter or something?

Reply:
We'll change this once we add the filter to the 5gsec org.